Automattic/wp-calypso
# Jetpack Plans: Remove Polldaddy

Fixes #11117 

We've stopped offering Polldaddy in Professional, so this PR removes it from the UI and from the autoconfig flow. To fully remove it from being installed, you'll also need D4843.

Polldaddy was still mentioned in the FAQ below the plans grid, and in a card on the My Plans page, this PR also removes that.

To test

1. Apply D4843 to your sandbox
2. You should not see Polldaddy in the FAQs on `/plans/$site`
3. Visit My Plans for a Jetpack Professional site, you should not see a Surveys & Polls card
4. Visit `/plugins/setup/$site`, you should not see Polldaddy install

@ebinnion I'm not sure if this affects your install-in-checkout work, you'll definitely want to test with D4843
